cfme/configure/configuration/region_settings.py

```diff
@@ -266,6 +278,20 @@ def delete(self, cancel=False):
         assert view.is_displayed
         view.flash.assert_no_error()
 
+    def import_tag_from_file(self, file_name):
+        """Import Tag from CSV formatted file
+
+            Args:
+                file_name: Name of .csv file containing tag data.
+                By default file must be at cfme/tests/configure/
```

Comment:
Why must the file be at this location? tempfile should take care of the file location, and it shouldn't matter where the file is for the import in the UI.

Reply:
Removed the comment. his is where I was seeing the framework put the file. I guess it handles this location so we don't need to care about it.